docs/modules/ROOT/pages/references/dns-naming-scheme.adoc

```diff
@@ -0,0 +1,99 @@
+= DNS naming scheme
+
+NOTE: The DNS naming scheme fulfills the usability requirements documented in xref:appuio-cloud:ROOT:references/quality-requirements/usability/dns-naming-scheme.adoc[DNS Naming scheme].
+
+== Base domain
+
+The base domain is `appuio.cloud`.
+
+Common services that are zone independent are served on the third level.
+Services bound to a specific {zone}, will be served through the fourth level where the third level is the zone's identifier.
+Apps served through am {zone}'s default domain use the subdomain `apps` of a zone's base domain.
+
+.Known hosts and schemes of 'appuio.cloud'
+|===
+| Hostname | Description
+
+|`appuio.cloud`, `www.appuio.cloud`
+| Hosting the marketing homepage for {product}
+
+|`id.appuio.cloud`
```

Comment:
Probably it would make sense to be more specific here. `keycloak.appuio.cloud` and a CNAME for specific protocols like OCID, SAML. This would it make more easy, if we might switch to a different solution than Keycloak in the future.

Reply:
Switching to another technology than Keycloak is a great architectural change. I don't think we should already prepare for that.
Adding CNAMEs like OCID and SAML would probably be purely technical implementation detail, not sure about the relevance in the architecture documentation.

Reply:
I'd rather not add such a complexity for a migh-be-probably-maybe feature.